lib/inbox/pages/inbox_page.dart

```diff
@@ -81,6 +81,15 @@ class _InboxPageState extends State<InboxPage> {
         centerTitle: false,
         title: AutoSizeText('Inbox', style: theme.textTheme.titleLarge),
         actions: [
+          IconButton(
+            icon: const Icon(
+              Icons.checklist,
+              semanticLabel: 'Read All',
```

Comment:
It looks like nothing else on this page is localized, so I wouldn't be a stickler about it, but it is nice to try to localize any new strings that we're adding. 😊

Reply:
Fixing this. Not clear on how you generate the localization files. I added `build_runner: ^2.4.6`

Reply:
Sorry, we need better documentation on localization!

1. Add your string to all `.arb` files.
2. Run `flutter gen-l10n`.
3. Replace the string with `AppLocalizations.of(context)!.yourResourceName`.

Let me know if that's enough detail! We should probably throw it in a README.

What does `build_runner` do?

Reply:
> What does build_runner do?


> provides general-purpose commands for generating files, including testing the generated files or serving both source and generated files. [Build runner](https://dart.dev/tools/build_runner)

E.g If you have localization files, generated models, it will gen all this in one go.